Server/port_discovery.py

```
@@ -158,3 +196,112 @@ def get_port_config() -> Optional[dict]:
+                project_name = PortDiscovery._extract_project_name(project_path)
+                port = data.get('unity_port')
+                is_reloading = data.get('reloading', False)
+
+                # Parse last_heartbeat
+                last_heartbeat = None
+                heartbeat_str = data.get('last_heartbeat')
+                if heartbeat_str:
+                    try:
+                        last_heartbeat = datetime.fromisoformat(heartbeat_str.replace('Z', '+00:00'))
+                    except Exception:
+                        pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)